frontend/lib/src/components/shared/InputInstructions/InputInstructions.tsx

```diff
@@ -22,23 +22,23 @@ import { StyledWidgetInstructions } from "@streamlit/lib/src/components/widgets/
 import { StyledMessage } from "./styled-components"
 
 export interface Props {
+  allowEnterToSubmit: boolean
```

Comment:
Nit: still slightly prefer the optional param with default true to avoid passing `allowEnterToSubmit={true}` in `ChatInput` and test file, but can understand if done for sake of clarity.